test/e2e/cert-manager_test.go

```diff
@@ -0,0 +1,72 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	certmanager "go.etcd.io/etcd-operator/pkg/certificate/cert-manager"
+	interfaces "go.etcd.io/etcd-operator/pkg/certificate/interfaces"
+)
+
+func TestCertManagerCM(t *testing.T) {
```

Comment:
It seems this e2e test isn't actually executed at all, because you do not execute it like below?

https://github.com/etcd-io/etcd-operator/blob/0049f0b76f5b53fdf873f0a3e64b340b9acb65ec/test/e2e/e2e_test.go#L62

Reply:
Also the e2e test is too simple, it should at least cover the following cases
- It should cover all interface methods
- It should verify the created certificate match the provided parameters

Reply:
Sure, I will update the tests accordingly.